### PR TITLE
In memoriam of OrdoM

### PR DESCRIPTION
**Pull request by optimumtact**
## Why It's Good For The Game
In Memoriam 26th August, 2020
Our friend, tg admin, and champion of the "short" reply, OrdoDictionary passed away recently.

While this news was not unexpected - Ordo had been fighting bravely against prion disease for some time - it remains as heartbreaking as it is unfair.

Ordo was sharp as a razor, with wit to match. During the time we knew him he was pursuing a PhD, yet he still found time to volunteer on the tgstation servers and help his fellow spessman.

In admin channels you were as likely to find Ordo pontificating on history or food or ham radio as you were anything related to the game. But you knew - no matter the topic - that his opinion was informed and valuable, and that he would deliver it with nuance, class and charm.

This community - and the world - is worse off without him in it.
He will be missed.

So should you read this and should you get the chance, raise a glass in his memory.
…but be sure to break out the good stuff.

That’s how he would have wanted it.

Adds a medal in memoriam of Ordo and adds it to the Head of Personnel locker in a lockbox, along with three Medals of Bureaucratic Excellence, for those who excel in managing the station.
 
Thanks to kryson for the sprite of the medal and thanks to OrdoM for all the work he did on tg.
## Changelog
:cl: oranges, kryson
add: Added a medal box for the Head Of Personnel
add: Added the OrdoM memorial medal for paperwork excellence
/:cl:
image of in game medal

<details>
  <summary>Admin stuff about memorial items</summary>
  Note, we don't usually add items to the game in response to external community events,  this is a rare exception to the norm, given the impact Ordo had on the /tg/station community. Uwnder normal circumstances we wold opt instead to update the comment on the Tomb of the Unknown Employee to memorialise those lost who had an impact on the community and game.
</details>

### CODE REVIEW

File: code/game/objects/structures/memorial.dm [hunk 1]
```diff
@@ -20,3 +19,27 @@ This memorial has been designed for him and any future coders to perish.
...
+no matter the topic - that his opinion was informed and valuable, and that he would deliver it with nuance, class and charm.
+
+This community - and the world - is worse off without him in it.
+He will be missed.
+
+So should you read this and should you get the chance, raise a glass in his memory.
+…but be sure to break out the good stuff.
+That’s how he would have wanted it.
+*/
+/obj/item/clothing/accessory/medal/gold/ordom
```
Comment (optimumtact) [inline code/game/objects/structures/memorial.dm:40]: I know this is technically misfiled, but I figured we'd keep these important objects separate from the rest of the code so there's no accidental deletions.

File: code/game/objects/structures/memorial.dm [hunk 2]
```diff
@@ -20,3 +19,27 @@ This memorial has been designed for him and any future coders to perish.
...
+	name = "The OrdoM Memorial Medal For Excellence in paperwork"
+	desc = "Awarded for outstanding excellence in paperwork, administration and bureaucracy"
+	icon_state = "medal_paperwork"
+	medaltype = "medal-gold"
+	custom_materials = list(/datum/material/gold=1000)
```
Comment (ATH1909) [inline code/game/objects/structures/memorial.dm:45]: You might not want to give this medal any custom_materials, or else irreverent people might deconstruct it for the gold it contains.

Alternatively, you could make it indestructable/undeconstructable like the captain's medal, which'd be the better option, IMO.

File: code/modules/clothing/under/accessories.dm [hunk 3]
```diff
@@ -197,6 +197,11 @@
 	medaltype = "medal-gold"
 	custom_materials = list(/datum/material/gold=1000)
 
+/obj/item/clothing/accessory/medal/gold/bureaucracy
+	name = "Excellence in Bureaucracy Medal"
+	desc = "Awarded for services to the Corporation in management"
```
Comment (ATH1909) [inline code/modules/clothing/under/accessories.dm:202]: should this have a period at the end?

File: code/game/objects/structures/memorial.dm (hunk 2 above)
Comment (optimumtact) [reply code/game/objects/structures/memorial.dm:45]: I don't think that's necessary, yes people might deconstruct that, but that is a minor risk and I'm not too worried about it.

File: code/modules/clothing/under/accessories.dm (hunk 3 above)
Comment (optimumtact) [reply code/modules/clothing/under/accessories.dm:202]: Yes I will do a grammar pass this evening.

File: code/game/objects/structures/memorial.dm [hunk 4]
```diff
@@ -20,3 +19,27 @@ This memorial has been designed for him and any future coders to perish.
...
+
+This community - and the world - is worse off without him in it.
+He will be missed.
+
+So should you read this and should you get the chance, raise a glass in his memory.
+…but be sure to break out the good stuff.
+That’s how he would have wanted it.
+*/
+/obj/item/clothing/accessory/medal/gold/ordom
+	name = "The OrdoM Memorial Medal For Excellence in paperwork"
```
Comment (ShizCalev) [inline code/game/objects/structures/memorial.dm:41]: capitalize paperwork

File: code/modules/clothing/under/accessories.dm [hunk 5]
```diff
@@ -197,6 +197,11 @@
 	medaltype = "medal-gold"
 	custom_materials = list(/datum/material/gold=1000)
 
+/obj/item/clothing/accessory/medal/gold/bureaucracy
+	name = "Excellence in Bureaucracy Medal"
```
Comment (ShizCalev) [inline code/modules/clothing/under/accessories.dm:201]: \improper

File: code/game/objects/structures/memorial.dm (hunk 4 above)
Comment (ShizCalev) [inline code/game/objects/structures/memorial.dm:41]: \improper

Comment (optimumtact) [reply code/game/objects/structures/memorial.dm:41]: where

**Review by Twaticus — APPROVED** [review]
:heart:

File: code/game/objects/structures/memorial.dm [hunk 6]
```diff
@@ -20,3 +19,27 @@ This memorial has been designed for him and any future coders to perish.
...
+This community - and the world - is worse off without him in it.
+He will be missed.
+
+So should you read this and should you get the chance, raise a glass in his memory.
+…but be sure to break out the good stuff.
+That’s how he would have wanted it.
+*/
+/obj/item/clothing/accessory/medal/gold/ordom
+	name = "\proper The OrdoM Memorial Medal For Excellence in Paperwork"
+	desc = "Awarded for outstanding excellence in paperwork, administration and bureaucracy."
```
Comment (ShizCalev) [inline code/game/objects/structures/memorial.dm:42]:
```suggestion
	desc = "Awarded for outstanding excellence in paperwork, administration, and bureaucracy."
```

File: code/modules/clothing/under/accessories.dm [hunk 7]
```diff
@@ -197,6 +197,11 @@
 	medaltype = "medal-gold"
 	custom_materials = list(/datum/material/gold=1000)
 
+/obj/item/clothing/accessory/medal/gold/bureaucracy
+	name = "\improper Excellence in Bureaucracy Medal"
+	desc = "Awarded for services to the Corporation in management"
```
Comment (ShizCalev) [inline code/modules/clothing/under/accessories.dm:202]:
```suggestion
	desc = "Awarded for exemplary managerial services rendered while under contract with Nanotrasen."
```

**Comment by TheVekter** [comment]
I'm... kind of surprised. I didn't think we'd do this for anyone, but if any member of our community was worth the effort, it was Ordo.

File: code/game/objects/structures/memorial.dm (hunk 6 above)
Comment (optimumtact) [reply code/game/objects/structures/memorial.dm:42]: I don't follow the oxford comma

**Pull request merged**